axelor-account/src/main/java/com/axelor/apps/account/web/AccountController.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2018 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.axelor.apps.account.web;
+
+import java.math.BigDecimal;
+
+import com.axelor.apps.account.db.Account;
+import com.axelor.apps.account.db.repo.AccountRepository;
+import com.axelor.apps.account.service.AccountService;
+import com.axelor.exception.service.TraceBackService;
+import com.axelor.i18n.I18n;
+import com.axelor.rpc.ActionRequest;
+import com.axelor.rpc.ActionResponse;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+@Singleton
+public class AccountController {
+
+  @Inject private AccountRepository accountRepository;
+
+  @Inject private AccountService accountService;
+
+  public void computeBalance(ActionRequest request, ActionResponse response) {
+    try {
+      Account account = request.getContext().asType(Account.class);
+      account = accountRepository.find(account.getId());
+
+      BigDecimal balance =
+          accountService.computeBalance(account, AccountService.BALANCE_TYPE_DEBIT_BALANCE);
+
+      if (balance.compareTo(BigDecimal.ZERO) >= 0) {
+        response.setAttr("$balance", "title", I18n.get("Debit balance"));
+      } else {
+        response.setAttr("$balance", "title", I18n.get("Credit balance"));
+      }
+
+      response.setValue("$balance", balance);
```

Comment:
Balance must be always positive as mentioned on ticket. So if balance is negative multiply it by '-1'